src/ResourceManager/OperationalInsights/Commands.OperationalInsights.Test/ScenarioTests/DataSourceTests.ps1

```diff
@@ -28,7 +28,7 @@ function Test-DataSourceCreateUpdateDelete
     New-AzureRmResourceGroup -Name $rgname -Location $wslocation -Force
 
     # Create a workspace to house the data sources
-    $workspace = New-AzureRmOperationalInsightsWorkspace -ResourceGroupName $rgname -Name $wsname -Location $wslocation -Force
+    $workspace = New-AzureRmOperationalInsightsWorkspace -ResourceGroupName $rgname -Name $wsname -Location $wslocation -Sku premium -Force
```

Comment:
This would require re-record of DataSourceCreateUpdateDelete.json, while I saw you just have re-record of CreateAllKindOfDataSource.json.